spec/database_helpers.rb

```diff
@@ -1,7 +1,7 @@
-# require 'pg'
-#
-# def persisted_data(id:)
-#   connection = PG.connect(dbname: 'chitter_test')
-#   result = connection.query("SELECT * FROM peeps WHERE id = #{id};")
-#   result.first
-# end
+require 'pg'
+
+def persisted_data(id:)
+  connection = PG.connect(dbname: 'chitter_test')
+  result = connection.query("SELECT * FROM peeps WHERE id = #{id};")
+  result.first
+end
```

Comment:
Looks like getting all the database files in there right at the start really helped. Would love to hear the details behind how you approached this.